hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerID.java

```diff
@@ -41,8 +41,8 @@
    * @param id int
    */
   private ContainerID(long id) {
-    Preconditions.checkState(id > 0,
-        "Container ID should be a positive. %s.", id);
+    Preconditions.checkState(id >= 0,
```

Comment:
We don't create a container with ID 0, but 0 is a valid ID.
The reason for allowing 0 as a value for container ID is to avoid the explicit null check that we do in HDDS-1302.